pkgs/development/libraries/libevent/default.nix

```diff
@@ -12,9 +12,10 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ python ];
 
-  patchPhase = ''
+  buildInputs = stdenv.lib.optional (python ? isPy2) python;
```

Comment:
two reasons:

1) it's confusing to check if it's python2, as what you're really checking is if it depends on python
2) it may support python3 some day